src/PdfSharper/Pdf.AcroForms/PdfSignatureField.cs

```diff
@@ -112,7 +110,6 @@ public PdfSignatureField(PdfDocument document)
             sign.Elements.Add(Keys.M, new PdfDate(DateTime.Now));
 
             document._irefTable.Add(sign);
-            document._irefTable.Add(this);
```

Comment:
PdfSignatureHandler.cs::AddSignatureComponents() is the only reference in the actual project. It grabs the Annotations array and adds this. Then it grabs the AcroForm's Fields and adds it to that as well. Do we need a separate add to the ireftable of the document? I thought those would take care of it. None of the other fields have explicit adding to the doc's table.